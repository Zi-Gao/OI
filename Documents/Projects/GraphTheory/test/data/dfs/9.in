626 460
345 174 6213
571 273 29330
218 214 12850
515 570 1795
71 355 15333
537 144 25509
74 469 8571
108 10 2829
284 186 20856
449 568 5022
195 376 29851
517 167 6340
72 468 5064
604 523 23429
566 459 15145
452 592 6886
460 489 21970
534 226 481
331 375 32648
202 120 19420
301 19 29316
167 18 16865
106 376 5441
402 286 16903
361 527 4282
129 594 28619
492 64 20896
355 602 18252
431 302 30581
548 314 26277
404 30 2700
341 54 18397
272 197 28253
262 116 11042
306 99 30635
180 508 25673
451 495 26232
286 512 28623
491 215 20918
245 411 29383
602 175 26412
385 567 26237
569 560 19481
490 360 31678
522 314 31602
466 44 2801
124 407 17614
490 531 32151
484 225 7004
23 316 21194
358 155 18042
154 30 966
18 399 19913
185 330 28337
341 86 14536
96 26 23829
288 130 32358
549 179 2247
384 360 23301
162 198 32052
393 584 17383
253 356 11339
236 526 25853
276 533 21886
473 569 4743
554 504 17034
617 576 27156
408 142 12556
185 424 5821
421 273 20748
521 370 23207
252 60 24575
593 93 13375
544 127 25851
133 285 16292
158 157 15881
1 396 25915
292 608 22486
107 259 3176
506 534 24637
444 175 30164
169 304 22871
521 351 8920
258 567 30231
87 533 22491
100 487 27297
175 352 26050
278 337 10702
485 207 2766
38 254 13997
171 591 30069
547 111 3033
205 254 21525
82 217 29686
471 287 29342
255 222 21951
99 24 17275
264 360 15202
608 254 25445
429 286 4131
481 20 16362
333 470 116
134 157 5831
224 475 26638
404 589 6250
561 273 5807
131 39 8757
556 493 16254
11 512 21852
168 463 28749
582 547 21870
404 468 17711
81 248 17906
373 394 544
300 490 16537
162 493 8105
543 512 1521
89 425 30919
514 623 10406
323 170 30602
126 573 30634
467 46 11029
361 600 1510
59 340 27924
152 532 28214
235 475 30142
372 114 26855
128 126 17657
283 271 1009
363 546 27715
139 387 24981
397 144 22629
201 487 22840
451 22 2847
12 146 17550
422 451 5196
560 290 14182
527 132 22322
63 94 27332
161 339 5109
245 265 21515
321 17 5741
10 468 18265
72 599 8903
493 295 12398
405 409 3518
377 602 28598
79 168 12626
415 597 20805
561 280 28733
473 357 29165
574 36 2464
192 182 19123
592 455 27544
72 18 5456
136 521 20157
496 210 23682
253 566 32188
95 104 24913
447 42 4996
216 373 6817
490 290 32708
537 582 1692
140 167 5291
506 424 24032
3 279 4646
384 111 1650
469 250 25105
178 620 30953
184 54 22042
185 329 13384
336 535 838
372 584 28033
230 413 8270
596 34 28064
279 74 25435
121 464 7680
221 594 4296
316 321 9254
433 400 4075
134 585 14094
460 27 15387
418 538 5646
22 41 5993
167 609 19871
70 123 20941
250 132 20652
461 110 4780
311 211 27953
86 303 27654
326 272 22868
590 610 9655
443 135 4675
382 50 29821
226 591 6259
51 625 973
562 292 19736
148 469 19104
231 371 25326
418 537 17076
179 534 2221
532 455 5890
350 229 15802
65 312 2377
124 510 12791
587 494 30728
615 157 14268
528 225 25924
370 399 15201
141 543 29654
348 409 30178
247 386 21580
554 351 14468
33 198 23415
84 194 19842
499 333 19517
177 154 3624
262 523 2892
457 589 2290
302 524 20152
166 103 29307
543 56 13495
499 195 12903
544 280 21484
105 379 14736
51 432 7431
349 218 274
198 625 6307
409 450 10778
222 463 32165
492 180 17528
399 587 7055
74 180 22120
347 206 29055
92 46 27153
327 595 21973
11 586 13731
80 190 7092
571 329 31012
41 373 31450
345 311 31047
497 92 29569
297 516 8418
543 481 19276
271 200 23263
472 94 24939
7 414 13504
343 388 18292
13 95 24069
38 68 6573
383 552 13081
563 142 19642
424 66 17048
250 481 24548
224 172 12803
551 404 27257
146 170 9022
413 323 8185
130 481 20080
52 251 31236
324 169 31772
507 63 2125
139 589 31113
82 342 17912
378 538 5588
20 298 4395
151 622 16454
436 431 19651
430 3 4287
313 167 7529
162 228 26961
80 235 27252
597 440 3179
375 125 8288
121 80 1309
497 413 30149
176 67 10071
524 284 29966
366 244 19521
18 616 7270
377 135 21229
150 5 21129
29 218 26649
38 554 2699
532 355 14272
32 199 2471
358 245 415
215 587 10679
368 387 9734
246 14 30699
617 39 12934
459 33 714
475 315 20056
444 427 23473
198 246 13464
192 414 3912
17 209 16040
489 534 30109
416 173 27473
3 382 1515
501 260 24450
171 129 9024
65 167 9147
465 198 28268
181 523 28038
33 249 24331
266 20 9867
475 440 2189
155 409 11045
327 108 20857
620 612 30844
68 399 21098
561 184 18074
557 212 24439
420 287 25836
228 312 14849
90 141 12331
230 174 25111
327 282 11956
374 120 17918
148 337 23713
214 38 29557
580 375 843
593 588 16843
286 401 12293
66 506 31976
171 125 3102
552 25 14551
525 146 3227
553 158 29293
279 182 14889
457 26 19749
482 162 16931
378 156 26448
151 199 10201
197 423 13592
581 146 22862
451 48 26651
503 504 11557
250 107 27773
67 509 30591
608 88 17859
494 401 491
214 229 25817
593 5 22211
600 369 15888
212 168 2695
255 197 16628
457 454 3601
435 48 11597
536 341 10938
157 121 26534
488 159 9670
184 9 13616
616 521 15032
557 116 31200
51 14 13402
526 426 1291
362 143 31378
429 315 30716
153 84 32615
273 531 12084
517 310 8134
487 7 8777
157 197 23772
195 525 14257
377 28 22820
496 574 24785
353 269 20605
116 151 18471
470 57 25827
340 470 15981
551 170 8606
271 582 13572
246 556 5379
346 376 17163
429 375 25888
254 483 16363
67 41 24037
254 274 14654
112 526 14251
200 378 17247
48 41 19240
107 322 11014
536 21 9260
570 143 13235
294 458 30405
392 301 8673
307 156 29329
444 513 8497
1 132 24173
421 55 30225
208 322 23235
610 488 10120
339 449 13090
157 344 14638
537 125 22500
185 463 30331
430 527 32531
158 361 2031
39 94 6090
116 179 24519
469 202 15805
312 622 20277
255 164 17082
419 277 5717
399 65 16958
427 523 20568
404 46 5993
484 538 2720
333 584 2044
213 245 28853
364 376 5935
474 221 4199
335 526 18806
576 50 8311
84 36 16013
451 182 14743
25 576 31081
260 498 25765
405 387 1505
367 221 4722
279 379 31270
305 469 20588
498 478 9831
568 537 11012
336 100 12082
270 428 31150
375 586 24435
71 554 6599
441 256 17695
17 246 13514
288 552 14317
561 355 25492
309 464 19369
208 542 9132
393 179 30732
150 464 29900
491 605 32276
585 220 25624
246 391 22358
209 494 6087
236 295 21956
506 617 26931
624 370 6871
558 528 5245
6 273 17181
114 13 19538
460 78 8096
347 292 25970
622 67 10397
354 421 8454
562 611 3458
537 540 11750
331 199 29750
471 233 6246
135 198 11787
591 534 23869
247 242 29425
236 448 32671
