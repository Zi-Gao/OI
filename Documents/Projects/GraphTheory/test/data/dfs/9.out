1 132 396 